app/assets/config/manifest.js

```diff
@@ -1,5 +1,13 @@
 //= link_tree ../images
 //= link_directory ../javascripts .js
 //= link_directory ../stylesheets .css
+//= link_directory ../stylesheets/codemirror/lib .css
+//= link_directory ../stylesheets/codemirror-buttons .css
```

Comment:
This is weird -- the space before the .css -- maybe it's just github?

Reply:
nope, definitely is a problem on my filesystem as well.

Reply:
I don't think this is a problem.

Reply:
Discussed and decided this was OK.